bitnami/discourse/templates/configmaps.yaml

```diff
@@ -10,20 +10,24 @@ metadata:
   annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
   {{- end }}
 data:
-  {{- $port:=.Values.service.port | toString }}
-  DISCOURSE_HOSTNAME: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
+  {{- $port := .Values.service.port | toString }}
+  DISCOURSE_HOST: "{{ include "discourse.host" . }}{{- if ne $port "80" }}:{{ .Values.service.port }}{{ end }}"
```

Comment:
The env var `DISCOURSE_HOSTNAME` is now `DISCOURSE_HOST` . Wouldn't it better leave it as `HOSTNAME` and minimize the number of changes for the user ? We already have `HOSTNAME` in a ton of places.

Reply:
There is an alias in place, so if users specify `DISCOURSE_HOSTNAME` it would still work